doc/src/manual/noteworthy-differences.md

```diff
@@ -224,6 +229,24 @@ For users coming to Julia from R, these are some noteworthy differences:
   * The commonly used `Int` type corresponds to the machine integer type (`Int32` or `Int64`).
     This means it will overflow, such that `2^64 == 0`. If you need larger values use another appropriate type,
     such as `Int128`, [`BigInt`](@ref) or a floating point type like `Float64`.
+  * The imaginary unit `sqrt(-1)` is represented in Julia as `im`, not `j` as in Python.
+  * In Julia, the exponentiation operator is `^`, not `**` as in Python.
+  * In Julia uses `nothing` of type `Nothing` to represent null value, whereas Python uses `None` of type `NoneType`.
+  * In Julia, the standard operators over a matrix type are matrix operations, whereas, in Python, the standard operators are element-wise operations. When both `A` and `B` are matrices, `A * B` in Julia performs matrix multiplication, not element-wise multiplication as in Python. `A * B` in Julia is equivalent with `A @ B` in Python, whereas `A * B` in Python is equivalent with `A .* B` in Julia.
+  * The adjoint operator `'` in Julia returns an adjoint of a vector (a lazy representation of row vector), whereas the transpose operator `.T` over a vector in Python returns the original vector (non-op).
+  * Functions in Python have one implementation (no polymorphism), whereas in Julia a function may contain multiple concrete implementations (called *Methods*), selected via multiple dispatch.
+  * There are no classes in Julia. Instead they are structures (mutable or immutable), containing data but no methods. 
+  * Calling a method of a class in Python (`a = MyClass(x), x.func(y)`) corresponds to a function call in Julia, e.g. `a = MyStruct(x), func(x::MyStruct, y)`. In general, multiple dispatch is more flexible and powerful than the Python class system.
+  * Python classes can inherit from one or more superclasses. Julia structures may have exactly one abstract supertype. 
+  * The Python code structure is defined by directories (Packages) and files (Modules), whereas the logical Julia program structure (Packages and Modules) is independent of the file strucutre (`include` for additional files).
+  * Conditional assignment in Python `x = 1 if y>0 else -1` corresponds to the Ternary operator `x = y>0 ? 1 : -1` in Julia.
+  * Macros in Julia can do what decorators in Python do, and much more.
```

Comment:
I think this is a harmful statement to make casual Python programmers think that macros and decorators are somewhat similar.  If we are to include this, I think we need to make it clear that macros operate on expression and decorators operate on function objects.  Maybe what you want to say is "decorator-looking things" like `@inline` in Julia are actual macros?

Reply:
The decorator syntax is much closer to `do` block in Julia actually.

Reply:
I had something like the `@time` macro in mind, which you could do with a decorator to the timed function in Python. But I agree that this may be misleading.
The suggestion to point out the different meaning of `@` is good, I'll change this point.